web_interface/alert_config_app/forms.py

```diff
@@ -215,13 +193,33 @@ def __init__(self,*args,**kwargs):
                 'class':'form-control',
                 'type':'text',
                 'placeholder':'dd hh:mm:ss',
-                #'data-toggle':'tooltip',
-                #'data-placement':'top',
-                #'title':'tooltip!',
             }
         )
     )
-    
+
+    def clean_new_owners(self):
+        """Validate the new set of owners
+
+        Returns
+        -------
+        [account_mgr_app.models.Profile]
+        """
+        pks = self.cleaned_data['new_owners']
+        if len(pks) == 0:
+            raise forms.ValidationError("Alerts must have at least one owner")
+        
+        pks = [int(pk) for pk in pks]
+
+        owners_list = []
+
+        for pk in pks:
+            try:
+                owners_list.append(Profile.objects.get(pk=pk))
+            except Exception as E:
+                print(type(E),type(E).__name__)
```

Comment:
Whats up with this `print`? You ever think about logging error messages like these?

Reply:
I'll set up real logging for the web_interface.